scrapy/core/downloader/contextfactory.py

```diff
@@ -54,7 +60,7 @@ def getCertificateOptions(self):
                         method=getattr(self, 'method',
                                        getattr(self, '_ssl_method', None)),
                         fixBrokenPeers=True,
-                        acceptableCiphers=DEFAULT_CIPHERS)
+                        acceptableCiphers=self.tls_ciphers)
```

Comment:
We should remove or deprecate the `DEFAULT_CIPHERS` import in this file, and probably also its definition in `scrapy/core/downloader/tls.py`.

Reply:
> We should remove or deprecate the `DEFAULT_CIPHERS` import in this file, and probably also its definition in `scrapy/core/downloader/tls.py`.

And move the default to `__init__`?

Reply:
I mean that `DEFAULT_CIPHERS`, after your change, is no longer used in this file, nor anywhere else in the Scrapy code base.

Reply:
And I am completely wrong about that :|